shared/patches/expo-image+1.8.0.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/node_modules/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt b/node_modules/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt
+index 75f5612..a665b38 100644
+--- a/node_modules/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt
++++ b/node_modules/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt
+@@ -33,7 +33,7 @@ import expo.modules.image.records.ImageProgressEvent
+ import expo.modules.image.records.ImageTransition
+ import expo.modules.image.records.SourceMap
+ import expo.modules.kotlin.AppContext
+-import expo.modules.kotlin.tracing.beginAsyncTraceBlock
++//import expo.modules.kotlin.tracing.beginAsyncTraceBlock
+ import expo.modules.kotlin.tracing.trace
+ import expo.modules.kotlin.viewevent.EventDispatcher
+ import expo.modules.kotlin.views.ExpoView
+@@ -265,7 +265,7 @@ class ExpoImageViewWrapper(context: Context, appContext: AppContext) : ExpoView(
+     // However, in this case, it is safe to use as long as nothing else is added to the queue.
+     // The intention is simply to wait for the Glide code to finish before the content of the underlying views is changed during the same rendering tick.
+     mainHandler.postAtFrontOfQueue {
+-      trace(Trace.tag, "onResourceReady") {
++      trace(Trace.tag) {
+         val transitionDuration = (transition?.duration ?: 0).toLong()
+ 
+         // If provided resource is a placeholder, but the target doesn't have a source, we treat it as a normal image.
+@@ -340,9 +340,9 @@ class ExpoImageViewWrapper(context: Context, appContext: AppContext) : ExpoView(
+         // If our image is animated, we want to see if autoplay is disabled. If it is, we should
+         // stop the animation as soon as the resource is ready. Placeholders should not follow this
+         // value since the intention is almost certainly to display the animation (i.e. a spinner)
+-        if (resource is Animatable && !isPlaceholder && !autoplay) {
```

Comment:
autoplay isn't a var here so i dunno how this would have worked at all. in the MR its a var above but maybe in this tag its half committed back?